frontend/src/utils/formatDate.ts

```diff
@@ -0,0 +1,10 @@
+export const formatDate = (date: string) => {
```

Comment:
I haven't looked at any of the rest of this PR and may not do so until we get the other one hashed out, but just because I see this now too -- we shouldn't make localization choices for users, the whole point of localization is that users should see the right thing for where they're from :) -- no one should have to see US-formatted dates if they don't generally use US-formatted dates, so we should be using the locale-aware ways to render dates.

Reply:
Something like

```
const options = {
  weekday: 'long',
  year: 'numeric',
  month: 'long',
  day: 'numeric',
};
yourDate.toLocaleDateString(undefined, options);
```

according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString.

Reply:
Fixed!